static/app/views/replays/detail/browserOSIcons.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled';
+
+import ContextIcon from 'sentry/components/replays/contextIcon';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import {Tooltip} from 'sentry/components/tooltip';
+import {space} from 'sentry/styles/space';
+
+export default function BrowserOSIcons() {
+  const {replay} = useReplayContext();
+  const replayRecord = replay?.getReplay();
+
+  return (
+    <IconContainer>
```

Comment:
nit:
we could replace this `IconContainer` with `Fragment` because the outer `ContextContainer` has the same gap and alignments. We should update the ContextContainer grid-columns to be `1fr max-content max-content` to guarantee that there are 3 columns, but as long as the first column is set to `1fr` it should look fine.